pinot-controller/src/main/java/com/linkedin/pinot/controller/api/ControllerRestApplication.java

```diff
@@ -153,6 +153,11 @@ public void handle(Request request, Response response) {
 
     final Redirector redirector = new Redirector(getContext(), "/swagger-ui/index.html?url=/api", Redirector.MODE_CLIENT_TEMPORARY);
     router.attach("/help", redirector);
+
+    final Directory landing = new Directory(getContext(), getClass().getClassLoader().getResource("landing").toString());
```

Comment:
better to check for resource being null? I know it cant be if classpath is set properly but hard to debug a NPE. I suggest adding a warning if the resource is not found and move on.